### ISSUE TITLE
I'm hoping to add one feature: a program packager

### ISSUE THREAD

**Issue by iamtornado** [PSAppDeployToolkit/PSAppDeployToolkit#777]
PSAppDeployToolkit is a great tool, and I'm hoping to add one feature: a program packager. Because I want to package the folder as a self-extracting exe file, the user can directly double-click this file to start the installation

**Comment by endoleg** [comment]
great idea!

**Comment by cyborgrd** [comment]
You can do it with WinRar. You can create an self extracting exe and execute one after

**Comment by iamtornado** [comment]
Yes, thanks! I also found that the same functionality can be implemented with 7zip。

**Comment by Ath3na-UK** [comment]
> It would be better if PSAppDeployToolkit integrated this packaging feature

It doesn't really need to, it takes 2 seconds to create your own using 7zip

**Comment by iamtornado** [comment]
Yes，I will try 7zip

**Comment by iamtornado** [comment]
Just Now ，I used PSAppDeployToolkit to create a script to uninstall mcafee, then I used 7zip to create a self-extracting exe file, found a problem, I found that Show-InstallationRestartPrompt did not work, after the program was executed normally, I did not see the restart prompt box, Show-BalloonTip worked fine.
I think this should have something to do with 7zip creating self-extracting exe files, possibly the exe process closing the powershell process created by Show-InstallationRestartPromp.
Is there a good way to solve this problem? I wish a restart prompt box would pop up.
